### ISSUE TITLE
Tooltip position issue in Firefox

### ISSUE THREAD

**Issue by vjo** [AnSavvides/d3act#20]
The tooltip has no position set, so it appear at the bottom of the chart in Firefox, see this screenshot:

![screen shot 2015-11-01 at 18 17 39](https://cloud.githubusercontent.com/assets/1239752/10870529/f9af9c42-80c4-11e5-99fa-ead89e937877.png)

By looking into the JS console when running the examples, I have this error:

``` js
ReferenceError: event is not defined                     app.js:29752:18
```

It comes from `BaseChart.js` [line 57](https://github.com/AnSavvides/d3act/blob/master/src/components/BaseChart.js#L57) and 58:

``` js
onMouseMove() {
    if (!this.showTooltips) {
        return;
    }

    const top = (event.pageY - 10); // line 57
    const left = (event.pageX + 10); // line 58
...
```

In short `window.event` doesn't exist in Firefox, the work around would be to use the `d3.event` instead.

**Comment by AnSavvides** [comment]
Ha, well spotted, I should probably do a browser audit! :)

**Comment by AnSavvides** [comment]
Fixed in `v1.1.1` :tada: 

<img width="644" alt="screen shot 2015-11-01 at 10 11 31 pm" src="https://cloud.githubusercontent.com/assets/2316326/10871680/8f543f94-80e5-11e5-81ea-ee235c06dbfd.png">
